gcc/config/aarch64/winnt.cc

```diff
@@ -36,6 +36,7 @@ along with GCC; see the file COPYING3.  If not see
 #include "options.h"
 #include "memmodel.h"
 #include "emit-rtl.h"
+#include "aarch64-protos.h"
```

Comment:
I am not sure about adding the include here as `i386/winnt.cc` do not include `i386-protos.h` but I haven't found how this header should be included into the `aarch64/winnt.cc` yet...